checks.d/wmi_check.py

```diff
@@ -109,8 +109,12 @@ def _extract_metrics(self, results, metrics, tag_by, wmi, tag_queries, constant_
                     # Special-case metric will just submit 1 for every value
                     # returned in the result.
                     val = 1
-                else:
+                elif getattr(res, wmi_property):
                     val = float(getattr(res, wmi_property))
+                elif not getattr(res, wmi_property):
```

Comment:
`else` instead of `elif not getattr(res, wmi_property) ` as `not getattr(res, wmi_property)` will always be `True` at this state.